cmd/booster-http/server.go

```diff
@@ -112,31 +110,31 @@ const idxPage = `
           Download a raw piece by payload CID
         </td>
         <td>
-          <a href="/payload/payloadcid">/payload/<payload cid></a>
+          <a href="/piece?payloadCid=bafySomePayloadCid&format=piece" > /payload/<payload cid></a>
```

Comment:
Let's also change the link target text for these:
```suggestion
          <a href="/piece?payloadCid=bafySomePayloadCid&format=piece" >/piece?payloadCid=<payload cid></a>
```